playground/frontend/playground_components/test/src/common/categories.dart

```diff
@@ -18,14 +18,16 @@
 
 import 'dart:collection';
 
-import 'package:playground_components/src/models/category_with_examples.dart';
-import 'package:playground_components/src/models/sdk.dart';
+import 'package:playground_components/playground_components.dart';
 
 import 'examples.dart';
 
 final categoriesMock = [
   CategoryWithExamples(title: 'Sorted', examples: [exampleMock1]),
-  CategoryWithExamples(title: 'Unsorted', examples: [exampleMock2]),
+  CategoryWithExamples(
+    title: 'Unsorted',
+    examples: [exampleMock1, exampleMock2, exampleMock2, exampleMock2],
```

Comment:
Are they repeated to test that 'tag2' is more frequent than 'tag1'? If so, explain in a comment please.